packages/core/src/components/select/select.tsx

```diff
@@ -74,6 +74,11 @@ export class Select {
    */
   @Prop() i18nSelectListHeader = 'Please select an option';
 
+  /**
+   * Hide list header
+   */
+  @Prop() selectListHeaderHide = false;
```

Comment:
We should rename this to _hideListHeader_

Reply:
Also @since is missing here